api/_globals/structuredClone.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/structured-data.html#dom-structuredclone",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "98"
           },
           "chrome_android": {
             "version_added": false
```

Comment:
Can you add support for Chrome on Android too? `structuredClone` is supported there since v98 too.